### PR TITLE
fix(init): fix webpack config scaffold in init package

### PR DESCRIPTION
**Pull request by anshumanv**
fix webpack config scaffold

ISSUES CLOSED: #1230

**What kind of change does this PR introduce?**
bugfix, sync lock files

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
NA

**Summary**
Function to generate webpack config file was being skipped, fixed it.

**Does this PR introduce a breaking change?**
Fixes instead.

**Other information**

### CODE REVIEW

File: packages/utils/modify-config-helper.ts [hunk 1]
```diff
@@ -130,8 +144,8 @@ export default function modifyHelperUtil(action: string, generator: typeof Gener
 
                 const successMessage = `\nYou can now run ${chalk.green(runCommand)} to bundle your application!\n\n`;
                 process.stdout.write(`\n${successMessage}`);
-                return;
```
Comment (anshumanv) [inline packages/utils/modify-config-helper.ts]: We were exiting before actually creating the config.

**Comment by anshumanv** [comment]
Lock files seem out of sync and need to be updated eventually, should I do that in a separate PR @ematipico ?

**Comment by ematipico** [comment]
That's strange because I don't get all these changes locally. Though, it would be better to do it on a separate PR

**Comment by anshumanv** [comment]
Sounds good! Will push another commit.

> That's strange because I don't get all these changes locally.

Can you try running npm run bootstrap? I got these when I did that I believe.

**Comment by ematipico** [comment]
OK no problem, we will soon switch to `yarn` because we need workspaces feature in order to test different packages together

**Pull request merged**